bravado/requests_client.py

```diff
@@ -139,7 +141,7 @@ def add_response_detail_to_errors(e):
         args = list(e.args)
         args[0] += (' : ' + e.response.text)
         e.args = tuple(args)
-    raise e
+    raise HTTPError(*e.args), None, sys.exc_info()[2]
```

Comment:
My wording could have been misinterpreted:
- unit test (implies testing the `if` conditional)
- docstrings because I didn't know the expected type of `e`